README.md

```diff
@@ -1 +1,118 @@
 # xtractmime
+
+`xtractmime` is a [BSD-licensed](https://opensource.org/licenses/BSD-3-Clause)
+Python 3.6+ implementation of the [MIME Sniffing
+Standard](https://mimesniff.spec.whatwg.org/).
+
+Install from [`PyPI`](https://pypi.python.org/pypi/xtractmime):
+
+```
+pip install xtractmime
+```
+
+---
+
+## Basic usage
+
+Below mentioned are some simple examples of using `xtractmime.extract_mime`:
+
+```python
+>>> from xtractmime import extract_mime
+>>> body = b"Sample text content"
+>>> extract_mime(body)
+b'text/plain'
+>>>
+```
+
+```python
+>>> from xtractmime import extract_mime
+>>> content_types = (b'text/html',)
+>>> body = b''
+>>> extract_mime(body, content_types=content_types)
+b'text/html'
+>>> 
+```
```

Comment:
Try to minimize code as long as it does not negatively affect readability. Also, keep a consistent style (e.g. same type of quotes for strings).

````suggestion
```python
>>> from xtractmime import extract_mime
>>> extract_mime(b'Sample text content')
b'text/plain'
>>> extract_mime(b'', content_types=(b'text/html',))
b'text/html'
```
````